content/guides/hot-module-replacement.md

```diff
@@ -94,6 +88,13 @@ __index.js__
   }
 
   document.body.appendChild(component());
++
```

Comment:
I think that's ok, it's just indicating that a new line was added in between the last line of the script and new content. That's how we've done it in the other guides as well.